### PR TITLE
Update content.md

### PR DESCRIPTION
**Pull request by akash73**
added section "how to change default password"

## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [X ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 1]
```diff
@@ -750,6 +750,17 @@ To gain admin (root) access, type `sudo su -` and the password, which by default
 
 You can now freely program your Portenta X8 Linux OS. In the sections below you can check some basic commands to get started.
 
+### Change default user password
+
+Your X8 comes with the default user fio with password fio.
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:755]:
```suggestion
Your Portenta X8 comes with the default user fio with password fio.
```

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 2]
```diff
@@ -750,6 +750,17 @@ To gain admin (root) access, type `sudo su -` and the password, which by default
...
+
+For security reasons, we strongly suggest to change the default password. To do so, when logged in into your X8, launch this command to change the
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:757]:
```suggestion
For security reasons, we strongly suggest changing the default password. To do so, when logged in to your Portenta X8, launch this command to change the password of the fio account:
```

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 3]
```diff
@@ -750,6 +750,17 @@ To gain admin (root) access, type `sudo su -` and the password, which by default
 
 You can now freely program your Portenta X8 Linux OS. In the sections below you can check some basic commands to get started.
 
+### Change default user password
+
+Your X8 comes with the default user fio with password fio.
+
+For security reasons, we strongly suggest to change the default password. To do so, when logged in into your X8, launch this command to change the 
+password of the fio account:
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:758]:
```suggestion
```

**Review by martab1994 — APPROVED** [review]
LGTM thanks :)

File: content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md [hunk 4]
```diff
@@ -750,6 +750,16 @@ To gain admin (root) access, type `sudo su -` and the password, which by default
 
 You can now freely program your Portenta X8 Linux OS. In the sections below you can check some basic commands to get started.
 
+### Change default user password
```
Comment (jhansson-ard) [inline content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md:753]:
```suggestion
### Change Default User Password
```

**Review by martab1994 — APPROVED** [review]
Thanks =)

**Pull request merged**